app/src/main/java/de/danoeh/antennapod/activity/VideoplayerActivity.java

```diff
@@ -61,12 +58,7 @@
 
     private final AtomicBoolean isSetup = new AtomicBoolean(false);
 
-    private LinearLayout controls;
-    private LinearLayout videoOverlay;
-    private AspectRatioVideoView videoview;
-    private ProgressBar progressIndicator;
-    private FrameLayout videoframe;
-    private ImageView skipAnimationView;
+    VideoplayerActivityBinding viewBinding;
```

Comment:
This variable is defined in both `VideoplayerActivity` and its parent class `MediaplayerActivity`. This could lead to confusion. Please remove it here. It then uses the parent one in the code below.